.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: false
+language: python
+python:
+- '3.6'
+install: pip install tox-travis
+script: tox
+notifications:
+  slack:
+    secure: aV2yV0GJrWzF2F6FPVG7WiKWkjYClBzWxr4BrojnBR8zj2CJ/MMMpZvAbma3KQkJx7cYGlr0XbCJ6vEnSfcVachyHlEbjWM95yQVxBhtfvmh8gldkSMyv950gqruNRb5uNkoKOfm6iY3vooHjWOOBLML9Xhb3E3T0qPQpgk7PSvCeZtkz5sWurGK+dalvCZBLmBkSSqJfXHDdOYpyPbg+WBLR8jkpmjVW1sVKFCTrNQooJUJQKrV//Xb4I/GDkL25078L2RptRrY/RLW/pewf9omg67NZ71mUqCCmnbbWTImYvJSEHSWumq31tRPUKWzZ9WV0Ors2HDpc2BcdJLl1qfLFkCUxcQzmRYYr3rlg4Mn4m6Jm5XN2nNE7ffiD42brINw/00lhzQuhnORcX8OP7eTa3EfFFopbbcE9PGclW4/saL3TtOz6A3ov613Ig/fVyfbPGy8Sx2PVlpIGIW0lHgw3gzqNTFuYqX1pnvgaE67xNhI8IznMdd9xM4hSwAV81I5Yzi6Bd6qEYYmDUfcOBGbeb1tJWsC/E57DxJL97/7dfkwWy/0E3qywY2+iS08RY0Y+oKStg8eC0eCO/Cqq1LnQgrYgLJEhxFBKo1kyL1g+Tv1+BGLe94HlnGbMBP886LTRVmiCjqvkn/k2B+PnMNuZvcWw/WUv64FtUGpw+o=
```

Comment:
This looks like something that shouldn't be hardcoded or public...

Reply:
It actually is fine, that secret was encrypted using travis and it is the recommended way, see https://docs.travis-ci.com/user/notifications/#Configuring-Slack-notifications.